test/integration/sanity/response-status-message.test.js

```diff
@@ -0,0 +1,64 @@
+var server = require('../../fixtures/servers/_servers'),
+    expect = require('chai').expect;
+
+describe('response status message', function () {
+    var testrun,
+        rawEchoServer;
+
+    before(function (done) {
+        var self = this;
+
+        rawEchoServer = server.createRawEchoServer();
+        rawEchoServer.listen(0, function () {
+            self.run({
+                collection: {
+                    item: [{
+                        request: {
+                            method: 'POST',
+                            url: rawEchoServer.url + '/raw-echo',
+                            body: {
+                                mode: 'raw',
+                                raw: 'HTTP/1.1 200 Working fine\r\n\r\n'
+                            }
+                        }
+                    }, {
+                        request: {
+                            method: 'POST',
+                            url: rawEchoServer.url + '/raw-echo',
+                            body: {
+                                mode: 'raw',
+                                raw: 'HTTP/1.1 200 Работает нормально\r\n\r\n'
+                            }
+                        }
+                    }]
```

Comment:
Add another test where it force encode a latin1 escaped character sequence.
Also, add a comment in the assertion explaining why it's force converted to utf8.